features/datachoices/src/main/java/org/opennms/features/datachoices/internal/UsageStatisticsReporter.java

```diff
@@ -74,6 +74,8 @@
 
     private AlarmDao m_alarmDao;
 
+    private boolean m_useSystemProxy = true; // true == legacy behaviour
```

Comment:
I believe it should be `m_useSystemProxy = false; // false == legacy behaviour`.

Reply:
@agalue if you look at line 131 (code on left side - before my change) you can see that clientWrapper.useSystemProxySettings(); was always called - therefor I think this is the legacy behavior. We could set `private boolean m_useSystemProxy = false;` this would make the behavior consistent with the other classes that use HttpClientWrapper.  That might be the better long term solution. What you you think? Shall I change it?